.github/workflows/helm_charts_scalar.yml

```diff
@@ -12,42 +12,69 @@ on:
     paths:
       - "charts/**"
 
+env:
+  HELM_VERSION: v3.5.2
+
 jobs:
   lint-chart:
     runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.7
+
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-      - name: Fetch history for chart testing
-        run: git fetch --prune --unshallow
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: ${{ env.HELM_VERSION }}
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.0.0-rc.2
+      - name: Set up Python
+        uses: actions/setup-python@v2
         with:
-          command: lint
-          config: .github/ct.yaml
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config .github/ct.yaml
 
   kubeval-chart:
     runs-on: ubuntu-latest
     needs: lint-chart
-    # Uses 1.16.4 since it doesn't support 1.16.13 in https://github.com/instrumenta/kubernetes-json-schema.
     strategy:
       matrix:
         k8s:
-          - v1.15.7
-          - v1.16.4
+          - v1.19.7
```

Comment:
I think it should be tested with multiple supported versions. But we haven't decided what versions to support. So, let's discuss and update.

Reply:
Updated k8s version in d9385c0, but can't find latest image.
 
- v.1.20.2 (latest)
https://hub.docker.com/r/kindest/node/tags?page=1&ordering=last_updated&name=1.20
- 1.18.15 (latest)
https://hub.docker.com/r/kindest/node/tags?page=1&ordering=last_updated&name=1.18